tasks/forms.py

```diff
@@ -6,7 +6,11 @@ class TaskForm(forms.ModelForm):
     def __init__(self, user_id, *args, **kwargs):
         super(TaskForm, self).__init__(*args, **kwargs)
         logged_user = User.objects.filter(user_id=user_id)
-        self.fields['assignee'].queryset = User.objects.filter(team=logged_user.first().team, role=Role.EMPLOYEE)
+        ass = User.objects.filter(team=logged_user.first().team, role=Role.EMPLOYEE)
```

Comment:
Could you change the variable name to something meaningful?

Reply:
Probably short variable name for assignee, still need to be changed.